sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -346,20 +348,28 @@ def test_explicit_set_string_index_type(self, client):
         poller.result()
 
     @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer(client_kwargs={"api_version": TextAnalyticsApiVersion.V3_1_PREVIEW_3})
+    @TextAnalyticsClientPreparer()
     def test_bidirectional_relation_type(self, client):
```

Comment:
if I understood the test correctly, the name should be changed too

Reply:
same comments in the async tests

Reply:
yeah i guess so, i'm always lazy to change the names. will do it, thanks!